### DISCUSSION TITLE
Change needed in Kong documentation for updation of Custom Entities

### DISCUSSION THREAD

**Discussion by narendrapatel** [Kong/kong#7616] [Help] [answered]
The update example in [custom-entities](https://docs.konghq.com/gateway-oss/2.5.x/plugin-development/custom-entities/#updating-an-entity) needs a change.

**Current**:
```
local entity, err = kong.db.keyauth_credentials:update({
  { id = "2b6a2022-770a-49df-874d-11e2bf2634f5" },
  { key = "updated_secret" },
})
```
**Proposed**:
```
local entity, err = kong.db.keyauth_credentials:update(
  { id = "2b6a2022-770a-49df-874d-11e2bf2634f5" },
  { key = "updated_secret" },
)
```

This is because [DAO:update](https://github.com/Kong/kong/blob/master/kong/db/dao/init.lua#L1134) function takes 2 mandatory arguments. The current example combines arguments and passes them as one argument.

Can some one please verify and confirm.

**Comment by kikito** [comment, marked as answer]
Thanks for reporting this, we'll amend the docs.

**Reply by kikito** [reply]
Here's the docs PR: https://github.com/Kong/docs.konghq.com/pull/3512

**Reply by narendrapatel** [reply]
Thank you @kikito :)
